README.Rmd

```diff
@@ -23,14 +23,57 @@ A shiny app for automated detection of signals in infectious disease surveillanc
 This app allows the user to investigate generated signals and corresponding surveillance time series. It includes upload of csv or Excel files, data checks, possibility to set parameters for signal detection, visualisation of results and generation of reports. The tool is interactive and flexible so that it can be used in different contexts and customised to the user’s needs.
 
 ## Installation
-
 You can install the development version of SignalDetectionTool from [GitHub](https://github.com/United4Surveillance/signal-detection-tool) with:
 
 ``` r
 # install.packages("devtools")
 devtools::install_github("United4Surveillance/signal-detection-tool")
 ```
-
+<details>
+  <summary><strong>Installing latest SignalDetectionTool version</strong></summary>
+  
+  <ul>
+    <li>You can download the latest release of the SignalDetectionTool through the Releases page</li>
+    <li>To navigate to the Release page, scroll up and look on the right-hand side of the page under a section labelled Releases. Click on 'Releases'</li>
+    <li>You can find the latest tool release at the top of the page</li>
+    <li>There are three possible ways to install the tool:</li>
+    <ul>
+      <li>If you just want to run the tool and do not want or need to see the code behind, you can follow the installation using .tar.gz</li>
+      <li>If you want to see the code behind the app in addition to running the tool, you can follow the installation from source after unzipping the zip file</li>
```

Comment:
SignalDetectionTool_x.x.x.zip: This is a binary. You can use this if you do not have Rtools installed and you are not interested in modifying the code of the app.